### ISSUE TITLE
Possible misprint in box.shard.eselect

### ISSUE THREAD

**Issue by derElektrobesen** [tarantool/shard#1]
`eselect` function takes a shard mode as first argument.
Then, if `shardno == me` condition failed, the request will be sent on the tarantool with number shardno, but with shard_mode of the current shard.

It is possible to change `mode` to `smode` in the following block:
```
  828             else
  829                 tnt = connection(shardno, smode)
  830             end
```

Master branch.

**Comment by Sulverus** [comment]
merged in https://github.com/tarantool/shard/commit/2fea15464d71eaf5d66c5bcbaba34eb776cc8c8c

**Comment by derElektrobesen** [comment]
Хорошо. Спокойной ночи.

2015-04-23 16:10 GMT+03:00 Andrew <notifications@github.com>:

> Closed #1 <https://github.com/tarantool/shard/issues/1>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tarantool/shard/issues/1#event-288294119>.
>



-- 
С уважением, Павел Бережной.
